src/pages/Patients/VerifyPatient.tsx

```diff
@@ -80,6 +80,18 @@ export default function VerifyPatient(props: { facilityId: string }) {
     enabled: !!patientData?.id && canListEncounters,
   });
 
+  const { data: closedEncounters } = useQuery({
```

Comment:
Why do you need it? 🤔

Reply:
We need to show, Open encounters from 1 month ago along with closed encounters, therefore the need for separate api calls